source/Calamari.Tests/KubernetesFixtures/ResourceStatus/Resources/PodTests.cs

```diff
@@ -28,10 +62,10 @@ public void WhenInitContainerIsBeingCreatedWithoutErrors_TheStatusShouldShowNumb
         {
             var podResponse = new PodResponseBuilder()
                 .WithPhase("Pending")
-                .WithInitContainerStatuses(new State[]
+                .WithInitContainerStates(new ContainerState[]
```

Comment:
Renamed some classes so they match the name of the entities in the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/).